### PR TITLE
Turned non-AD material property into AD propertiy 

### PR DESCRIPTION
**Pull request by tophmatthews**
ref #13494

### CODE REVIEW

**Comment by tophmatthews** [comment]
ping @dschwen ?

**Comment by dschwen** [comment]
Looks like a straight forward change, but the test is not illustrating the usefulness of this, as the coefficient is still just a constant :-/

**Comment by tophmatthews** [comment]
☹️

File: modules/tensor_mechanics/test/tests/ad_viscoplasticity_stress_update/lps_dual.i [hunk 1]
```diff
@@ -65,10 +82,11 @@
     relative_tolerance = 1e-11
   [../]
   [./coef]
-    type = ParsedMaterial
+    type = ADParsedMaterial
     f_name = coef_3
     # Example of creep power law
-    function = '0.5e-18 * exp(-4e4 / 1.987 / 1200)'
+    args = temp
+    function = '0.5e-18 * exp(-4e4 / 1.987 / temp)'
```
Comment (tophmatthews) [inline modules/tensor_mechanics/test/tests/ad_viscoplasticity_stress_update/lps_dual.i:89]: Updated one of the inputs @dschwen to show coupling

**Comment by tophmatthews** [comment]
ping @dschwen

**Review by dschwen — APPROVED** [review]

**Pull request merged**